richie/settings.py

```diff
@@ -13,6 +15,39 @@
 DATA_DIR = os.path.join("/", "data")
 
 
+def get_release():
+    """Get current release of the application
+
+    By release, we mean the latest commit's sha1. Two strategies are tested to
+    get this info:
+
+    1. from the version.json file à la Mozilla [1] (if any)
+    2. from the git repository (if any)
+
+    If none of those returns the release, it defaults to "NA".
+
+    [1] https://github.com/mozilla-services/Dockerflow/blob/master/docs/version_object.md
+    """
+    project_dir = os.path.abspath(os.path.join(BASE_DIR, os.pardir))
+
+    # Try to get the current release from the version.json file generated by the
+    # CI during the Docker image build
+    try:
+        with open(os.path.join(project_dir, "version.json")) as version:
+            return json.load(version)["commit"]
+    except FileNotFoundError:
```

Comment:
Shall we handle malformed json ?

Reply:
Good remark. I don't think so as it is auto-generated by the CI.